### PR TITLE
feat: Introduce Prisma on Node

### PR DESCRIPTION
**Pull request by razvand**
Introduce Prisma to be deployed on KraftCloud. It uses binary compatibility mode (i.e. the `base` image).

Add:

* `Kraftfile`: build / run rules, including pulling the `node` image
* `Dockerfile`: Prisma filesystem
* `README.md`: document how to use

### CODE REVIEW

File: node-prisma/Kraftfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spec: v0.6
+
+name: node-prisma
+
+runtime: index.unikraft.io/official/node:latest
```
Comment (nderjung) [inline node-prisma/Kraftfile:5]:
```diff
- runtime: index.unikraft.io/official/node:latest
+ runtime: node:18
```

File: node-prisma/Kraftfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+spec: v0.6
+
+name: node-prisma
```
Comment (nderjung) [inline node-prisma/Kraftfile:3]:
```diff
- name: node-primsa
+ name: node18-prisma
```

File: node-prisma/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    git clone https://github.com/prisma/prisma-examples --depth=1; \
+    cd prisma-examples/javascript/rest-express; \
+    npm install; \
+    npx prisma migrate dev --name init \
+    ;
+
+FROM scratch
+
+# rest-express + Prisma
+COPY --from=build /src/prisma-examples/javascript/rest-express /rest-express
```
Comment (nderjung) [inline node-prisma/Dockerfile:24]: Can you place this into `/app`?

File: node-prisma/Kraftfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+runtime: index.unikraft.io/official/node:latest
+
+rootfs: ./Dockerfile
+
+cmd: ["/usr/local/bin/node", "/rest-express/src/index.js"]
```
Comment (nderjung) [inline node-prisma/Kraftfile:9]:
```diff
- cmd: ["/usr/local/bin/node", "/rest-express/src/index.js"]
+ cmd: ["/usr/bin/node", "/app/src/index.js"]
```

**Comment by nderjung** [comment]
In addition can you please rename this to `node18-prisma`?

File: node-prisma/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,24 @@
+FROM --platform=linux/x86_64 debian:bookworm AS build
```
Comment (nderjung) [inline node-prisma/Dockerfile:1]:
```diff
- FROM --platform=linux/x86_64 debian:bookworm AS build
+ FROM node:current-bookworm AS build
```

**Review by craciunoiuc — APPROVED** [review]
All good here. Thanks!

Reviewed-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>
Approved-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

**Pull request merged**